src/main.zig

```diff
@@ -3322,6 +3322,46 @@ const AfterUpdateHook = union(enum) {
     update: []const u8,
 };
 
+// we take the raw compile log text, split it back up, and prettify it
+fn printCompileLog(comp: Compilation, ttyconf: std.debug.TTY.Config) !void {
+    const log_text = comp.getCompileLogText();
+    if (log_text.len == 0) return;
+
+    var bw = io.bufferedWriter(io.getStdErr().writer());
```

Comment:
(If we go with version 2)
Btw, I noticed there's no write buffering for `renderToStdErr` (where it prints compile errors).
Maybe it's be possible to reuse the `io.BufferedWriter` I use for stderr here for `renderToStdErr` too at some point? Or is it already buffered and I'm missing something?